pkgs/development/python-modules/blspy/dont_fetch_dependencies.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/python-bindings/CMakeLists.txt b/python-bindings/CMakeLists.txt
+index 255e3bb..5f99c3a 100644
+--- a/python-bindings/CMakeLists.txt
++++ b/python-bindings/CMakeLists.txt
+@@ -6,8 +6,7 @@ include(FetchContent)
+ 
+ FetchContent_Declare(
+   pybind11
+-  GIT_REPOSITORY https://github.com/pybind/pybind11.git
+-  GIT_TAG        v2.6.2
++  SOURCE_DIR @pybind11_src@
```

Comment:
these patches will probably be a pain to maintain. Probably would be a good idea to ask upstream to allow for `pkg-config` to be added to allow for easy detection of installed libraries.

However, this shouldn't block the PR.